libs/dh/shared/assets/src/favicon/favcicon - Datahub.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M23 16.2798C23 20.0764 22.1548 22.9815 20.4643 24.9952C18.7738 26.9984 16.3389 28 13.1598 28H8V5H13.7023C16.6375 5 18.9209 5.98586 20.5526 7.95759C22.1842 9.92932 23 12.7034 23 16.2798ZM19.8209 16.4056C19.8209 10.8995 17.7561 8.14637 13.6266 8.14637H11.0151V24.8379H13.1598C17.6005 24.8379 19.8209 22.0271 19.8209 16.4056Z" fill="#05A388"/>
+<path d="M30.0159 14.7652C29.9966 14.8365 29.9378 14.9374 29.7614 15.0523C29.5816 15.1694 29.3092 15.2804 28.93 15.37C28.1722 15.5491 27.0927 15.6193 25.7521 15.567C23.0793 15.4628 19.4805 14.8771 15.5966 13.8271C11.7126 12.777 8.30897 11.4696 5.94774 10.2127C4.76349 9.5824 3.86636 8.97781 3.30211 8.44129C3.01973 8.1728 2.84032 7.93963 2.74405 7.74788C2.6496 7.55976 2.64961 7.44299 2.66887 7.37173C2.68814 7.30048 2.74697 7.19961 2.92334 7.08471C3.10311 6.96759 3.37558 6.85659 3.75478 6.76696C4.51251 6.58785 5.59207 6.51767 6.93261 6.56993C9.60546 6.67412 13.2042 7.25985 17.0882 8.30991C20.9721 9.35997 24.3758 10.6674 26.737 11.9242C27.9213 12.5546 28.8184 13.1591 29.3826 13.6957C29.665 13.9642 29.8444 14.1973 29.9407 14.3891C30.0351 14.5772 30.0351 14.694 30.0159 14.7652Z" stroke="#9C9E9F"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8 8.53639L11.0151 9.36901V8.14637H13.6266C16.2913 8.14637 18.0963 9.29274 19.0415 11.5855L22.6534 12.5829C22.2704 10.7291 21.5702 9.18729 20.5526 7.95759C18.9209 5.98586 16.6375 5 13.7023 5H8V8.53639Z" fill="#05A388"/>
+<circle cx="20" cy="15" r="3" fill="#FED425"/>
+</svg>
```

Comment:
suggestion: rename this file to "Original-DataHub-Favicon.svg" and move it just outside the "assets" folder so we don't include it in the bundle.